app/src/utils/Fips.ts

```diff
@@ -1,9 +1,61 @@
 // TODO maybe rename this file since this string is out of place
 export const ALL_RACES_DISPLAY_NAME = "All races";
-export const USA_DISPLAY_NAME = "United States";
+export const USA_DISPLAY_NAME = "the United States";
```

Comment:
Why did you add "the"? Most dashboards I've seen just use "United States", I think it's a little cleaner.

At the very least capitalize "the"

Reply:
katrina specifically requested this change... we can always change this later again as a group

Reply:
Sure I don't care too much, but can you capitalize "The"?